data-sets-api-server/src/main/java/org/zowe/unix/files/services/zosmf/ZosmfUnixFilesService.java

```diff
@@ -54,4 +54,10 @@ public String getUnixFileChtag(String path) {
         GetUnixFileChtagRunner runner = new GetUnixFileChtagRunner(path);
         return runner.run(zosmfConnector);
     }
+    
+    @Override
+    public void deleteUnixFileContent(String path, boolean isRecursive) {
```

Comment:
Name needs to be generalised for deleting files and directories

Reply:
in my view, unix file makes sense for both directory and file in this "delete" scenario